README.md

```diff
@@ -205,13 +215,15 @@ The build requires both `python3` and `cmake`. If you don't have them installed,
 
 ```
 brew install python
+brew install pipx
 brew install cmake
 ```
 
 Now install `aqt`, a command line package manager for `Qt`:
 
 ```
-pip3 install aqtinstall --break-system-packages
+sudo pipx install aqtinstall
```

Comment:
sudo doesn't sound correct.

Also, why standard pip isn't enough?

Reply:
Pip will break the system package. The message displayed suggests the use of pipx.

pipx would not install for me on a new Ubuntu install without using sudo.

Reply:
but why do you even pass `--break-system-packages` here? brew-installed python basically runs in its own sandbox and doesn't touch the system packages.

this line is for macOS, for Linux you can do whatever you want :)

Reply:
I can test it again, but it would not install without --break-system-packages

Reply:
well, I don't see that https://github.com/miurahr/aqtinstall recommends such flag

I suggest to check that you're really trying to use brew's python/pip and not system's (`which pip`)

Reply:
So uninstalled `brew` completely to test this. Brew's python includes `pip` and I made sure it was in my path.

But I still get this output:

```
➜  ~ pip3 install aqtinstall
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try brew install
    xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a Python library that isn't in Homebrew,
    use a virtual environment:
    
    python3 -m venv path/to/venv
    source path/to/venv/bin/activate
    python3 -m pip install xyz
    
    If you wish to install a Python application that isn't in Homebrew,
    it may be easiest to use 'pipx install xyz', which will manage a
    virtual environment for you. You can install pipx with
    
    brew install pipx
    
    You may restore the old behavior of pip by passing
    the '--break-system-packages' flag to pip, or by adding
    'break-system-packages = true' to your pip.conf file. The latter
    will permanently disable this error.
    
    If you disable this error, we STRONGLY recommend that you additionally
    pass the '--user' flag to pip, or set 'user = true' in your pip.conf
    file. Failure to do this can result in a broken Homebrew installation.
    
    Read more about this behavior here: <https://peps.python.org/pep-0668/>

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

[notice] A new release of pip is available: 24.0 -> 24.2
[notice] To update, run: python3.12 -m pip install --upgrade pip
```

Reply:
However, after running a fresh install of brew, I can install `pipx` and `aqtinstall` without `sudo`.

Reply:
Thanks! Now I think I get it: this is a change in 3.12, previously it was never needed.

Reply:
Cool